res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -207,67 +200,6 @@ limitations under the License.
         color: $secondary-content;
     }
 
-    &.mx_ThreadView .mx_EventTile {
-        // handling for hidden events (e.g reactions) in the thread view
-
-        &:not([data-layout=bubble]) {
-            &:hover.mx_EventTile_verified.mx_EventTile_info .mx_EventTile_line,
-            &:hover.mx_EventTile_unverified.mx_EventTile_info .mx_EventTile_line,
-            &:hover.mx_EventTile_unknown.mx_EventTile_info .mx_EventTile_line {
-                padding-inline-start: 0; // Override
-            }
-        }
-
-        &.mx_EventTile_info {
```

Comment:
Moved to https://github.com/matrix-org/matrix-react-sdk/pull/8689/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R871